flask_resty/filtering.py

```diff
@@ -351,7 +351,7 @@ def filter_query(self, query, view):
         :return: The filtered query
         :rtype: :py:class:`sqlalchemy.orm.query.Query`
         """
-        args = flask.request.args
+        args = view.request_args if view.args_schema else flask.request.args
```

Comment:
Reason being that caller might have some custom manipulation of the query args that could then get picked up by the filters